mozci/console/commands/push.py

```diff
@@ -587,6 +588,20 @@ def handle(self) -> None:
                     f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent or marked as caused by another push</comment>"
                 )
 
+            for task in p.tasks:
+                for classification, note in task.classifications:
```

Comment:
I think only `GroupSummary` objects have the `classifications` property

```
  AttributeError

  'Task' object has no attribute 'classifications'
```

Reply:
Fixed in c071b96.